src/components/ComparisonFilters/ComparisonFilters.js

```diff
@@ -58,6 +58,14 @@ const ComparisonFilters = ({
       setFilterState({ ...filterState, comparisonPointTwo: { id: activeFilters.comparisonPointTwo?.[0] } });
     }
 
+    if (!activeFilters.comparisonPointOne && filterState.comparisonPointOne?.name) {
+      setFilterState({});

```

Comment:
Hi M, sorry but this is the same problem as we talked about on the call. We cannot just set filter state to empty like this, you need to be more clinical about which piece of filterState you're trying to empty.